src/main/java/fi/fmi/avi/model/sigmet/AirmetReference.java

```diff
@@ -1,6 +1,5 @@
 package fi.fmi.avi.model.sigmet;

```

Comment:
The `AirmetReference` seem to be equal to `SigmetReference`. Could this class be removed and replaced with only `SigmetReference` (perhaps with a more general name, if possible)? If not, in other words there is a reason to have two different types, then maybe having a common superinterface declaring common properties? Anyway, the unused generic type should be removed at minimum.